app/helpers/form_helper.rb

```diff
@@ -6,4 +6,70 @@ def app_setting_value_field_for(app_setting, f)
       f.input :value
     end
   end
+
+  def us_states_territories
+    [
+      ['', ''],
+      ['Alabama', 'AL'],
+      ['Alaska', 'AK'],
+      ['American Samoa', 'AS'],
+      ['Arizona', 'AZ'],
+      ['Arkansas', 'AR'],
+      ['Armed Forces Americas', 'AA'],
+      ['Armed Forces Others', 'AE'],
+      ['Armed Forces Pacific', 'AP'],
+      ['California', 'CA'],
+      ['Colorado', 'CO'],
+      ['Connecticut', 'CT'],
+      ['Delaware', 'DE'],
+      ['District of Columbia', 'DC'],
+      ['Florida', 'FL'],
+      ['Georgia', 'GA'],
+      ['Guam', 'GU'],
+      ['Hawaii', 'HI'],
+      ['Idaho', 'ID'],
+      ['Illinois', 'IL'],
+      ['Indiana', 'IN'],
+      ['Iowa', 'IA'],
+      ['Kansas', 'KS'],
+      ['Kentucky', 'KY'],
+      ['Louisiana', 'LA'],
+      ['Maine', 'ME'],
+      ['Maryland', 'MD'],
+      ['Massachusetts', 'MA'],
+      ['Michigan', 'MI'],
+      ['Minnesota', 'MN'],
+      ['Mississippi', 'MS'],
+      ['Missouri', 'MO'],
+      ['Montana', 'MT'],
+      ['Nebraska', 'NE'],
+      ['Nevada', 'NV'],
+      ['New Hampshire', 'NH'],
+      ['New Jersey', 'NJ'],
+      ['New Mexico', 'NM'],
+      ['New York', 'NY'],
+      ['North Carolina', 'NC'],
+      ['North Dakota', 'ND'],
+      ['Northern Mariana Islands', 'MP'],
+      ['Ohio', 'OH'],
+      ['Oklahoma', 'OK'],
+      ['Oregon', 'OR'],
+      ['Pennsylvania', 'PA'],
+      ['Puerto Rico', 'PR'],
+      ['Rhode Island', 'RI'],
+      ['South Carolina', 'SC'],
+      ['South Dakota', 'SD'],
+      ['Tennessee', 'TN'],
+      ['Texas', 'TX'],
+      ['United States Minor Outlying Islands', 'UM'],
+      ['Utah', 'UT'],
+      ['Vermont', 'VT'],
+      ['Virgin Islands', 'VI'],
+      ['Virginia', 'VA'],
+      ['Washington', 'WA'],
+      ['West Virginia', 'WV'],
+      ['Wisconsin', 'WI'],
+      ['Wyoming', 'WY']
+    ]
+  end
```

Comment:
great question. I asked here: https://18f.slack.com/archives/ruby/p1466632124000169 yesterday and the consensus was do it yourself. I'm game for spinning this into a gem, esp since 18F/USDS would probably re-use it.